src/models/__tests__/community_business.test.integration.ts

```diff
@@ -35,7 +35,7 @@ describe('Community Business Model', () => {
     });
 
     test('get :: filter orgs by ID | non-existent ID resolves to empty array', async () => {
-      const orgs = await CommunityBusinesses.get(knex, { where: { id: 200 } });
+      const orgs = await CommunityBusinesses.get(knex, { where: { organisationId: 200 } });
```

Comment:
We can still unambiguously call this `id`. `organisationId` is verbose and adds no value. Remember, the point of the models is to present a convenient API that is decoupled from the database implementation.